### PR TITLE
update oncn.io to kmesh.net

### PR DESCRIPTION
**Pull request by lec-bit**
update the oncn.io package in the code repository to kmesh.net, and replace the dependencies and other related text.

fix https://github.com/kmesh-net/kmesh/issues/72

### CODE REVIEW

**Comment by lec-bit** [comment]
image

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module oncn.io/mesh
+module kmesh.net/mesh
```
Comment (hzxuzhonghu) [inline go.mod:1]: @lec-bit should align with the repo name `kmesh.net/kmesh`

**Review by hzxuzhonghu — COMMENTED** [review]
Otherwise lgtm

**Comment by lec-bit** [comment]
image

**Pull request merged**